deployment/tensorrt/main.cpp

```diff
@@ -267,7 +320,7 @@ std::vector<Detection> YOLOv5Detector::detect(cv::Mat& image) {
   int32_t num_detections = 0;
   std::vector<float> detection_boxes;
   std::vector<float> detection_scores;
-  std::vector<int> detection_labels;
+  std::vector<float> detection_labels;
```

Comment:
The class output of `EfficientNMS_TRT` is int32 now, check https://github.com/NVIDIA/TensorRT/tree/main/plugin/efficientNMSPlugin#outputs for more details.